README.md

```diff
@@ -0,0 +1,212 @@
+<p align="center">
+    <img src="images/banner.png" height="150" alt="'The Owly' bookstore">
+</p>
+
+# Hi there 👋
+
+Welcome to the world of online book shopping, where stories come to life with just a click!
+📚 This project isn't just any online bookstore, it's a dynamic platform crafted with Spring Boot
+magic, ready to whisk you away on a literary adventure.
+
+## Description
+
+The Online Bookstore API isn't just a solution, it's a gateway to a realm of endless literary
+possibilities. 🌟 Seamlessly blending cutting-edge web technologies with industry best practices,
+this project sets the stage for a smooth and secure online book shopping experience.
+
+### Tech Stack:
+
+* **Spring Boot**: Your trusty steed for crafting Java-based web applications.
+* **Spring Security**: Guardian of the gates, ensuring only authorized users enter.
+* **Spring Data JPA**: Your faithful guide through the labyrinth of database interactions.
+* **Spring Web**: Your ticket to the world of web-based applications with Spring MVC.
+* **Maven**: Your organizational maestro, orchestrating dependencies with finesse.
+* **Hibernate**: The sorcerer's apprentice, wielding ORM magic.
+* **Liquibase**: The architect of database transformations, sculpting data with precision.
+* **MySQL**: Your sturdy stronghold, guarding precious data.
+* **MapStruct**: The cartographer, charting the territories of object mapping.
+* **JUnit**: Your trusty sword in the battle for code quality.
+* **Mockito**: The shape-shifter, conjuring illusions for seamless testing.
+* **Docker**: Your vessel for encapsulating the essence of your application.
+* **Testcontainers**: Your genie in a bottle, conjuring ephemeral environments for testing prowess.
+
+### Features:
+
+* User Registration and Authentication: Join the literary community with ease and security.
+* Book Browsing and Searching: Embark on a quest through the vast library of literature.
+* Shopping Cart Management: Gather your literary treasures with finesse.
+* Order Processing: Seal the deal and bring your chosen tales home.
+* User Profile Management: Customize your literary journey with personalized settings.
+* Secure RESTful API Endpoints: Fortified endpoints, ensuring safe passage for data travelers.
+
+## Installation
+
+1. Clone the repository:
+    ```bash
+   git clone https://github.com/nklimovych/book-store.git
+   cd book-store
+   ```
+2. Configure the database: Create a new MySQL database:
+    ```sql
+    CREATE DATABASE bookstore;
+    ```
+   Update the `application.properties` file with your MySQL database credentials:
+    ```properties
+    spring.datasource.url=jdbc:mysql://localhost:3306/bookstore
+    spring.datasource.username=root
+    spring.datasource.password=password
```

Comment:
can we use ENV variables here as well? SInce we already have a `.env` file anyway